rfcs/20221124-monorepo.md

```diff
@@ -0,0 +1,179 @@
+↖️ Table of Contents
+
+# RFC: Monorepo
+
+**Status:** 🚧 WIP, comments are welcome nonetheless
+
+## Reviewers
+
+- [ ] &lt;your name here&gt;
+- [ ] &lt;your name here&gt;
```

Comment:
```suggestion
- [ ] @zackkrida 
- [ ] @sarayourfriend 
```

Does this reviewer pairing seem okay? My rationale is that @sarayourfriend perhaps works across the most repositories the most often, and also authored some of the more critical actions related to releasing and deploying. And myself because for now I've been the most involved in the planning of projects.

Reply:
I like this. But I do want to stress that this affects everyone so I'd appreciate if everyone took at least one look (no comments necessary).

Reply:
Definitely, I think it's very reasonable to expect everyone to read and consider the proposal.